.github/workflows/automated-release-workflow.yml

```diff
@@ -75,14 +75,14 @@ jobs:
         docker images
 
     - name: Update Operator Image name in Release Folder
-      uses: jacobtomlinson/gha-find-replace@master
+      uses: jacobtomlinson/gha-find-replace@b76729678e8d52dadb12e0e16454a93e301a919d
```

Comment:
The [latest release ](https://github.com/jacobtomlinson/gha-find-replace/releases) appears to have the SHA starting with  `f485fdc`?

Reply:
The master branch has the latest commit `b76729678e8d52dadb12e0e16454a93e301a919d`
![image](https://user-images.githubusercontent.com/69360493/153522429-83ef1606-ff6e-427d-96e5-5c7ee3d9fce3.png)